### DISCUSSION TITLE
Planning to Deprecate/Retire Features

### DISCUSSION THREAD

**Discussion by rangav** [thunderclient/thunder-client-support#1381] [Announcements]
### Attach Env to Collection
- How many users are using this feature? Now we have collection and request level variables using scripting, is the Attach Env to collection still useful?
<img width="800" alt="Screenshot 2023-10-19 at 19 37 38" src="https://github.com/rangav/thunder-client-support/assets/8637550/428afcb3-d3df-496a-b6e6-24a76c208987">

### External Script Files 
- Now we introduced Inline Scripting, do you still need external script files feature?
<img width="800" alt="Screenshot 2023-10-19 at 20 19 08" src="https://github.com/rangav/thunder-client-support/assets/8637550/d0138a4c-aef4-4e4d-b112-afb4bda2506b">

Please let us know your feedback

**Comment by DarinClarkSE** [comment]
Please don't remove the External Scripts feature. I use that heavily.

**Comment by rangav** [comment]
@DarinClarkSE Thanks for the feedback, We plan to support both options - `Inline Scripting` and `External JS files`
